docs/data-sources/appliances.md

```diff
@@ -0,0 +1,39 @@
+---
```

Comment:
@nikchern, this page needs a template file. The template file should be located in **templates/data-sources/appliances.md.tmpl** The example usage is added to the template. Otherwise, the example will disappear the next time someone does `go generate`

Take a peek at application_profile's [template file](https://github.com/spectrocloud/terraform-provider-spectrocloud/blob/main/templates/data-sources/application_profile.md.tmpl) for an example.

Reply:
noted, I will make a change.

Reply:
@nikchern this step was missed 😞